src/liger_kernel/chunked_loss/fused_linear_preference.py

```diff
@@ -0,0 +1,107 @@
+import torch
+
+
+class LigerFusedLinearPreferenceBase(torch.autograd.Function):
+    @staticmethod
+    def forward(
```

Comment:
general suggestion:

1. can we make loss_fn a member method in the base class and raise not implemented exception and the derive class needs to implement it? instead of being a separate python function

2. can we use `__super__` instead of `LigerFusedLinearPreferenceBase.forward` (backward). Or we can not do that for static method.


I will merge this PR first and we can address later! Great contribution!!